### ISSUE TITLE
Installing errors

### ISSUE THREAD

**Issue by kostonstyle** [joefitzgerald/go-plus#120]
npm http GET https://registry.npmjs.org/async
npm http GET https://registry.npmjs.org/fs-plus
npm http GET https://registry.npmjs.org/emissary
npm http GET https://registry.npmjs.org/glob
npm http GET https://registry.npmjs.org/temp
npm http GET https://registry.npmjs.org/underscore-plus
npm WARN `git config --get remote.origin.url` returned wrong result (git://github.com/joefitzgerald/atom-message-panel.git) 
npm http 304 https://registry.npmjs.org/glob
npm http 304 https://registry.npmjs.org/fs-plus
npm http 304 https://registry.npmjs.org/emissary
npm http 304 https://registry.npmjs.org/async
npm http 304 https://registry.npmjs.org/underscore-plus
npm http 304 https://registry.npmjs.org/temp
npm ERR! git clone git://github.com/joefitzgerald/atom-message-panel.git Cloning into bare repository 'C:\Users\bma\AppData\Roaming\npm-cache\_git-remotes\git-github-com-joefitzgerald-atom-message-panel-git-2ac45c06'...
npm ERR! git clone git://github.com/joefitzgerald/atom-message-panel.git fatal: unable to connect to github.com:
npm ERR! git clone git://github.com/joefitzgerald/atom-message-panel.git github.com[0: 192.30.252.128]: errno=No error
npm ERR! Error: Command failed: Cloning into bare repository 'C:\Users\bma\AppData\Roaming\npm-cache\_git-remotes\git-github-com-joefitzgerald-atom-message-panel-git-2ac45c06'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.252.128]: errno=No error
npm ERR! 
npm ERR! 
npm ERR!     at ChildProcess.exithandler (child_process.js:648:15)
npm ERR!     at ChildProcess.emit (events.js:98:17)
npm ERR!     at maybeClose (child_process.js:756:16)
npm ERR!     at Process.ChildProcess._handle.onexit (child_process.js:823:5)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "c:\\Users\\bma\\AppData\\Local\\atom\\app-0.165.0\\resources\\app\\apm\\node_modules\\atom-package-manager\\bin\\\\node.exe" "c:\\Users\\bma\\AppData\\Local\\atom\\app-0.165.0\\resources\\app\\apm\\node_modules\\atom-package-manager\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "c:\\Users\\bma\\AppData\\Local\\atom\\app-0.165.0\\resources\\app\\apm\\node_modules\\atom-package-manager\\.apmrc" "--userconfig" "C:\\Users\\bma\\.atom\\.apmrc" "install" "C:\\Users\\bma\\AppData\\Local\\Temp\\d-11505-8080-1m26mds\\package.tgz" "--target=0.18.0" "--arch=ia32" "--msvs_version=2013"
npm ERR! cwd C:\Users\bma\AppData\Local\Temp\apm-install-dir-11505-8080-wp6qd4
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.4
npm ERR! code 128
npm

**Comment by joefitzgerald** [comment]
```
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.252.128]: errno=No error
```

Check your internet connection. Make sure you can clone Guthub repos.

**Comment by kostonstyle** [comment]
I check git clone statement everything is fine

PS D:\> git clone https://github.com/joefitzgerald/go-plus
Cloning into 'go-plus'...
remote: Counting objects: 1919, done.
remote: Total 1919 (delta 0), reused 0 (delta 0)
Receiving objects: 100% (1919/1919), 1.87 MiB | 574.00 KiB/s, done.
Resolving deltas: 100% (1277/1277), done.
Checking connectivity... done.

**Comment by kostonstyle** [comment]
I try with apm install go-plus

PS D:\> apm install go-plus
Installing go-plus to C:\Users\bma\.atom\packages failed

npm http GET https://registry.npmjs.org/async
npm http GET https://registry.npmjs.org/emissary
npm http GET https://registry.npmjs.org/fs-plus
npm http GET https://registry.npmjs.org/glob
npm http GET https://registry.npmjs.org/temp
npm http GET https://registry.npmjs.org/underscore-plus
npm WARN `git config --get remote.origin.url` returned wrong result (git://github.com/joefitzgerald/atom-message-panel.g
it)
npm http 304 https://registry.npmjs.org/glob
npm http 304 https://registry.npmjs.org/async
npm http 304 https://registry.npmjs.org/temp
npm http 304 https://registry.npmjs.org/underscore-plus
npm http 304 https://registry.npmjs.org/emissary
npm http 304 https://registry.npmjs.org/fs-plus
npm ERR! git clone git://github.com/joefitzgerald/atom-message-panel.git Cloning into bare repository 'C:\Users\bma\AppD
ata\Roaming\npm-cache\_git-remotes\git-github-com-joefitzgerald-atom-message-panel-git-2ac45c06'...
npm ERR! git clone git://github.com/joefitzgerald/atom-message-panel.git fatal: unable to connect to github.com:
npm ERR! git clone git://github.com/joefitzgerald/atom-message-panel.git github.com[0: 192.30.252.130]: errno=No error
npm ERR! Error: Command failed: Cloning into bare repository 'C:\Users\bma\AppData\Roaming\npm-cache\_git-remotes\git-gi
thub-com-joefitzgerald-atom-message-panel-git-2ac45c06'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.252.130]: errno=No error
npm ERR!
npm ERR!
npm ERR!     at ChildProcess.exithandler (child_process.js:648:15)
npm ERR!     at ChildProcess.emit (events.js:98:17)
npm ERR!     at maybeClose (child_process.js:756:16)
npm ERR!     at Process.ChildProcess._handle.onexit (child_process.js:823:5)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "c:\\Users\\bma\\AppData\\Local\\atom\\app-0.165.0\\resources\\app\\apm\\node_modules\\atom-package-man
ager\\bin\\\\node.exe" "c:\\Users\\bma\\AppData\\Local\\atom\\app-0.165.0\\resources\\app\\apm\\node_modules\\atom-packa
ge-manager\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "c:\\Users\\bma\\AppData\\Local\\atom\\app-0.165.0\\res
ources\\app\\apm\\node_modules\\atom-package-manager\\.apmrc" "--userconfig" "C:\\Users\\bma\\.atom\\.apmrc" "install" "
C:\\Users\\bma\\AppData\\Local\\Temp\\d-11505-6688-tgx3q7\\package.tgz" "--target=0.18.0" "--arch=ia32" "--msvs_version=
2013"
npm ERR! cwd C:\Users\bma\AppData\Local\Temp\apm-install-dir-11505-6688-1hubwyc
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.4
npm ERR! code 128
npm

**Comment by joefitzgerald** [comment]
Fixed in `v3.0.12` - I switched to git+https.
